api/models/site.js

```diff
@@ -34,6 +44,22 @@ const beforeValidate = (site) => {
   }
 };
 
+
+function isEmptyOrUrl(value, throws = true) {
+  const validUrlOptions = {
+    require_protocol: true,
+    protocols: ['http', 'https'],
```

Comment:
Maybe we should change this to `['https']`? That is what the frontend validation is looking for. If we do that. We can also drop the `http` domain in the SQS test.